cli/Sources/TuistHasher/TargetScriptsContentHasher.swift

```diff
@@ -64,4 +84,63 @@ public struct TargetScriptsContentHasher: TargetScriptsContentHashing {
         }
         return try contentHasher.hash(stringsToHash)
     }
+
+    // MARK: - Private
+
+    private func processInputFileListPath(
+        _ fileListPath: AbsolutePath,
+        sourceRootPath: AbsolutePath,
+        pathsToHash: inout [AbsolutePath],
+        stringsToHash: inout [String]
+    ) async {
+        if fileListPath.pathString.contains("$") {
+            stringsToHash.append(fileListPath.relative(to: sourceRootPath).pathString)
+            Logger.current.notice(
+                "The path of the file \'\(fileListPath.url.lastPathComponent)\' is hashed, not the content. Because it has a build variable."
```

Comment:
```suggestion
                "The path of the file \'\(fileListPath.url.lastPathComponent)\' is hashed, not the content because it has a build variable."
```